static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1077,13 +1077,107 @@ $quote-mark: 35px;
 }
 // *************** Feature showcase overrides ***************
 
+// Articles can have the feature tone applied, but it's not always primary tone
 .has-feature-showcase-element {
+    .tonal__header > .u-cf {
+        .content__labels--not-immersive {
+            display: none;
+        }
+        //@include mq($until: leftCol){
+        margin-left: -1px;
+        padding-left: 1px;
+        margin-right: 1px;
+        // }
+        position: relative;
+        z-index: 3; //put over feature showcase
+
+        width: gs-span(3);
+        background: #ffffff;
+        margin-top: gs-height(2) * -1;
+        @include mq(mobileLandscape) {
+            width: gs-span(4);
+        }
+        @include mq(tablet) {
+            width: gs-span(6);
+        }
+        @include mq(leftCol) {
+            background: #ffffff;
+            margin-top: 0;
+            position: absolute;
+            top: 10px;
```

Comment:
Any chance you could use `gs-baseline`? See here for grid: https://github.com/guardian/guss-grid-system